nextflow.config

```diff
@@ -254,8 +299,27 @@ profiles {
     test_nothing            { includeConfig 'conf/test_nothing.config' }
     test_motus              { includeConfig 'conf/test_motus.config' }
     test_krakenuniq         { includeConfig 'conf/test_krakenuniq.config' }
+    test_malt               { includeConfig 'conf/test_malt.config' }
+    test_falco              { includeConfig 'conf/test_falco.config' }
+    test_fastp              { includeConfig 'conf/test_fastp.config' }
+    test_adapterremoval     { includeConfig 'conf/test_adapterremoval.config' }
+    test_bbduk              { includeConfig 'conf/test_bbduk.config' }
+    test_prinseqplusplus    { includeConfig 'conf/test_prinseqplusplus.config' }
```

Comment:
I've noticed some duplication in the test_profile.
May I recommend for all tests (except test_full), to include test.config, and the tag so `malt` for example, and in `test_malt.config` only include the params that needs changing.

Reply:
I opted actually to do this on purpose to minimise run time. 
Test would run for all profilers, but these separate ones turns off most of them and leaves a couple (all major phases of the pipeline are covered)

That said, we could still do something similar and instead make a `test_fast` which we import/include as the 'base' profile that we they modify

Reply:
I fully agree on the multiple test profiles so that we can test multiple stuff, but you have lots of duplication there that could be removed.

Reply:
I can fix it next release!